PatchKit/Classes/Extensions/UIView+Extensions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIView+Extensions.swift
+//  PatchKit
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import Foundation
+
+extension UIView {
+    
+    public func removeAllConstraints() {
```

Comment:
Could you just add some docs as to how this differs from `self.removeConstraints(self.constraints)` Just copy from  https://stackoverflow.com/a/30491911